### PR TITLE
Update meeting templates and add new ones

### PR DESCRIPTION
**Pull request by dcwalk**
- Moves templates to new folder
- Removes preceding '-' (not needed)
- Updates existing templates
- Adds new templates
- On merge:
  - [ ] update shortlinks 
    - https://link.hypha.coop/template
    - https://link.hypha.coop/wg-gov-template
    - https://link.hypha.coop/wg-template

### CODE REVIEW

File: templates/board.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+breaks:  false # See: https://github.com/hackmdio/codimd/issues/40#issuecomment-172927690
+tags: Hypha
+---
+# 2021-MM-DD Hypha Worker Co-op Board Meeting - Mmmm DD, 202X
+
+Time:       5:00-6:00pm ET  
+Location:   https://link.hypha.coop/calls  
+Attending:  ...  
+Cultivator: ...  
+Notetaker:  ...
```
Comment (patcon) [inline templates/board.md:11]: Thoughts on adding a choice emoji to front of each line here? Def no block, but I do love like these little guys, and happy to use quirky Hypha-style ones :) Also, happy to spawn a new ticket and not clog this one! Feels like an opp for a little bit of personality when ppl cruise our notes

File: templates/retreat.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+|   |   |   |
+|   |   |   |
+|   |   |   |
+
+
+### Introductions
+
+_Cultivator: X_ [If needed]
+_Scribe: X_ [If needed]
+_Timekeeper: X_ [If needed]
```
Comment (patcon) [inline templates/retreat.md:58]:
```suggestion
_Cultivator: ..._ [If needed]
_Scribe: ..._ [If needed]
_Timekeeper: ..._ [If needed]
```

Would keeping our "..." notation be ok in this doc? More search-able as well

Comment (patcon) [reply templates/retreat.md:58]: I can backfill if you feel samesies

**Review by patcon — APPROVED** [review]
LGTM. Left comments, but you're fine to merge at will

File: templates/planning.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Attending:  ...  
+Cultivator: ...  
+Notetaker:  ... 
+
+## Agenda
+
+_Length: 2 hours_  
+_Format: In person|Video call_
+
+_Cultivator: X_ [If needed]
```
Comment (dcwalk) [inline templates/planning.md:18]:
```suggestion
_Cultivator: ..._ [If needed]
```

File: templates/planning.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+_Scribe: X_ [If needed]
```
Comment (dcwalk) [inline templates/planning.md:19]:
```suggestion
_Scribe: ..._ [If needed]
```

File: templates/planning.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+_Timekeeper: X_ [If needed]
```
Comment (dcwalk) [inline templates/planning.md:20]:
```suggestion
_Timekeeper: ..._ [If needed]
```

File: templates/retreat.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+_Time Required: X hours_ 
+_When to do: Before X_
+
+During the retreat you're encouraged to share a few thoughts from the following readings:
+
+
+## Retreat Day X [or Session X]  
+
+_Length: 3 hours_
```
Comment (dcwalk) [inline templates/retreat.md:43]:
```suggestion
_Length: X hours_  
```

File: templates/members.md [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+    - 🏅 I value how [member-owner] ...
+    - 🏅 I respect the way that [member-owner] ...
+    - 🏅 ... [your own]
+
+
+## Session III - Strategy Session
+
+_Length: 1.5 hours_
+
+_Cultivator: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:156]:
```suggestion
_Cultivator: ..._ [If needed]
```

File: templates/members.md [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+_Scribe: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:157]:
```suggestion
_Scribe: ..._ [If needed]
```

File: templates/members.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+_Timekeeper: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:158]:
```suggestion
_Timekeeper: ..._ [If needed]
```

File: templates/members.md [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+[LUNCH BREAK]
+
+---
+
+
+## Session II  - Key Governance Decisions (aka “Board meeting”)
+
+_Length: 1 hour_
+
+_Cultivator: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:119]:
```suggestion
_Cultivator: ..._ [If needed]
```

File: templates/members.md [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+_Scribe: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:120]:
```suggestion
_Scribe: ..._ [If needed]
```

File: templates/members.md [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+_Timekeeper: X_ [If needed]
```
Comment (dcwalk) [inline templates/members.md:121]:
```suggestion
_Timekeeper: ..._ [If needed]
```

File: templates/members.md [hunk 13]
```diff
@@ -0,0 +1,208 @@
...
+Indicate when you have completed the above on [**Loomio**](#): [https://loomio.hypha.coop/#](#)
+
+
+## Session I  - Year in Review and Key Business (aka Members Meeting)
+
+_Length: 3 hours_
+
+Chair: Dawn
+Timekeeper: Benedict
+Scribe: Patcon
```
Comment (dcwalk) [inline templates/members.md:68]:
```suggestion
_Timekeeper: ..._ [If needed]
```

File: templates/members.md [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+
+Indicate when you have completed the above on [**Loomio**](#): [https://loomio.hypha.coop/#](#)
+
+
+## Session I  - Year in Review and Key Business (aka Members Meeting)
+
+_Length: 3 hours_
+
+Chair: Dawn
+Timekeeper: Benedict
```
Comment (dcwalk) [inline templates/members.md:67]:
```suggestion
_Scribe: ..._ [If needed]
```

File: templates/members.md [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+1. Complete the "[**How prepared do you feel for the AGM?**](#)" check-in. 
+
+Indicate when you have completed the above on [**Loomio**](#): [https://loomio.hypha.coop/#](#)
+
+
+## Session I  - Year in Review and Key Business (aka Members Meeting)
+
+_Length: 3 hours_
+
+Chair: Dawn
```
Comment (dcwalk) [inline templates/members.md:66]:
```suggestion
_Cultivator: ..._ [If needed]
```

**Pull request merged**